packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -225,17 +239,39 @@ const handleBrowserChoice = async (browser) => {
 
 const props = defineProps<{
   gql: TopNavFragment,
-  showBrowsers?: boolean
+  showBrowsers?: boolean,
+  forceOpenDocs?: boolean
+}>()
+
+const emit = defineEmits<{
+  (e: 'clearForceOpen'): void,
 }>()
 
-const docsMenuVariant: Ref<'main' | 'orchestration' | 'ci'> = ref('main')
+const docsMenuVariant: Ref<DocsMenuVariant> = ref('main')
 
-const promptsEl: Ref<HTMLElement | null> = ref(null)
+const promptsEl: Ref<ComponentPublicInstance | null> = ref(null)
+
+watch(() => props.forceOpenDocs, (newVal) => {
+  if (newVal === true) {
+    docsMenuVariant.value = 'ci1'
+  } else {
+    docsMenuVariant.value = 'main'
+  }
+}, {
+  immediate: true,
+})
```

Comment:
Can consider `watchEffect` which is like `watch` and `immediate: true` by default. It will automatically collect all reactive dependencies and re-run when they change. Could also be a ternary:

```ts
watchEffect(() => {
  docsMenuVariant.value = props.forceOpenDocs ? 'ci1' : 'main'
})
```

Reply:
Nice, I haven't used watchEffect yet, updated.